packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -360,15 +361,24 @@ const PersonalDetailsForm = observer(() => {
                                             name='phone'
                                             id={'phone'}
                                             label={localize('Phone number*')}
+                                            className={
+                                                account_settings?.phone_number_verification?.verified
+                                                    ? 'account-form__fieldset--phone'
+                                                    : ''
+                                            }
                                             //@ts-expect-error type of residence should not be null: needs to be updated in GetSettings type
                                             value={values.phone}
                                             onChange={handleChange}
                                             onBlur={handleBlur}
                                             required
                                             error={errors.phone}
-                                            disabled={isFieldDisabled('phone')}
+                                            disabled={
+                                                isFieldDisabled('phone') ||
+                                                !!account_settings?.phone_number_verification?.verified
+                                            }
```

Comment:
if verfied, BE should give it in  immutable fields  .This should be handled in BE. isnt?

Reply:
Yeappp I'll remove the logic for `account_settings?.phone_number_verification?.verified` 🙏